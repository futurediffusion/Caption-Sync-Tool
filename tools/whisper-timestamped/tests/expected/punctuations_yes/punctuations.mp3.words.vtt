WEBVTT

WEBVTT

00:00.380 --> 00:01.100
Dis-moi,

00:01.280 --> 00:01.500
est-ce

00:01.500 --> 00:01.660
que

00:01.660 --> 00:02.040
l'avion

00:02.040 --> 00:02.760
vole ?

