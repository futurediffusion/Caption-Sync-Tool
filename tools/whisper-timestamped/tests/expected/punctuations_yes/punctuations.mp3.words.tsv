start	end	text
380	1100	Dis-moi,
1280	1500	est-ce
1500	1660	que
1660	2040	l'avion
2040	2760	vole ?
